zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -194,14 +194,27 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
             );
         });
 
-        // Verify transfer
-        let transfer = TransferGadget::construct(
-            cb,
-            caller_address.expr(),
-            callee_address.expr(),
-            value.clone(),
-            &mut callee_reversion_info,
-        );
+        // Verify transfer only for CALL opcode.
+        let transfer = cb.condition(is_call.expr(), |cb| {
```

Comment:
this will meet nested condition not support issue. because  `TransferGadget::construct(` internally use condition already.

Reply:
I see. I remember that issue. But it seems that it is not reported when I run `cargo test -- callop_ -- --nocapture`.
Please let me check the code again. Thanks.